### ISSUE TITLE
Adjust messages in ~entitlementcard-cert channel

### ISSUE THREAD

**Issue by michael-markl** [digitalfabrik/entitlementcard#614]
Currently, the cert channel gets spammed a lot.
Reduce the verbosity in this channel.

**Comment by michael-markl** [comment]
The verbosity has been reduced by @maxammann by just sending the exit status to ~entitlementcard-cert. 🎉 Maybe we want to have some summary of the imports nevertheless. It's (very) low prio.

**Comment by maxammann** [comment]
I didnt know this task exists. Were we annoyed at the same day? 😃

**Comment by michael-markl** [comment]
yup :D my reaction was to write an issue, yours was to fix the problem :D

**Comment by maxammann** [comment]
Best would be to move this into grafana. I suggest we get an alert if no new import happened for more than 7+1 days.

**Comment by f1sh1918** [comment]
if an import fails we get a message in the cert channel but not for success.
Furthermore we get info about new version installed on staging and production.

We can keep this few important information in that channel
